packages/cli/src/commands/init/init.ts

```diff
@@ -90,7 +90,8 @@ const bumpYarnVersion = async (silent: boolean, root: string) => {
     if (yarnVersion) {
       // `yarn set` is unsupported until 1.22, however it's a alias (yarnpkg/yarn/pull/7862) calling `policies set-version`.
       const setVersionArgs =
-        yarnVersion.major > 1 && yarnVersion.minor >= 22
+        (yarnVersion.major > 1 && yarnVersion.minor >= 22) ||
+        yarnVersion.major >= 2
```

Comment:
Why are we not checking for specifically `yarnVersion.major === 1 && yarnVersion.minor >= 22`? would avoid this extra v2 check

Reply:
Make sense, I guess there's no one using Yarn 0.x 😅